client/services/FeedbackGeneratorService.js

```diff
@@ -20,13 +20,15 @@
 
 tie.factory('FeedbackGeneratorService', [
   'FeedbackObjectFactory', function(FeedbackObjectFactory) {
-    var jsToHumanReadable = function(jsVariable) {
+    var _jsToHumanReadable = function(jsVariable) {
       if (typeof jsVariable === 'string') {
         return '"' + jsVariable + '"';
       } else if (typeof jsVariable === 'number') {
         return String(jsVariable);
       } else if (typeof jsVariable === 'boolean') {
         return jsVariable ? 'True' : 'False';
+      } else if (Array.isArray(jsVariable)) {
+        return '[' + jsVariable + ']';
```

Comment:
Hmm, you might have to assemble the stringified elements recursively, actually, and then join them with ', ' (note the space). That would be much more future-proof and would correctly handle e.g. lists of bools.